src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
```

Comment:
Don't use the `body` tag for styling. Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.